src/lib/UserConfig.ts

```diff
@@ -0,0 +1,11 @@
+import fs = require("fs");
+import os = require("os");
+
+const homedir = os.homedir();
+const configModule = `${homedir}/.newsletter-cli/config`;
+
+export let userConfig = {} as any;
+if (fs.existsSync(`${configModule}.js`)) {
```

Comment:
could you create a class UserConfig here that would have proper types? The class would read & parse the actual config .js file. Also, please move reading the config file out of the global scope and inject the instance of UserConfig to the specific commands in index.ts.

Reply:
I've refactored to add a `UserConfig` class, but I'm not sure how you're wanting the injection to work from `index.js` -- are you wanting the instantiated `userConfig` object passed down through all object instantiations to the class that will eventually use it?

In that case, for example, the customizations for the `prepare` command would need to be passed to the creation of the `PrepareCommand` class and the `Newsletter` class.

Reply:
Yes, passing it via constructor will be appropriate.